### PR TITLE
[#95512034] Faster docker node deployment

### PR DESCRIPTION
**Pull request by mtekel**
Do the main and slow docker server installation in parallel on all servers. Then run the concurrent-ssh-bug affected actions in serial, node-by-node. This speeds up deployment of the platform considerably.

### CODE REVIEW

File: common.yml [hunk 1]
```diff
@@ -3,11 +3,6 @@
   sudo: yes
   roles:
     - bennojoy.redis
-
-- hosts: "{{ hosts_prefix }}-tsuru-db"
-  sudo: yes
-  sudo_user: root
-  roles:
```
Comment (dcarley) [inline common.yml]: The commit message doesn't mention why this changes. Which probably also means that it should be in a separate commit.

File: common.yml [hunk 2]
```diff
@@ -38,12 +33,16 @@
     - name: restart gandalf
       service: name=gandalf-server state=restarted
 
+# Configure all docker nodes in parallel
```
Comment (dcarley) [inline common.yml]: We should omit this comment because it's describing Ansible's default behaviour.

File: common.yml [hunk 3]
```diff
@@ -38,12 +33,16 @@
...
 - hosts: "{{ hosts_prefix }}-tsuru-docker*"
-  serial: 1
   sudo: yes
   roles:
     - docker_server
 
+# Run this serially due to concurrent ssh issue
```
Comment (dcarley) [inline common.yml]: If we're going to comment this then it should be the line above `serial: 1` to make it clear what the difference is and make it more likely that we remove the comment at the same time as the line.

**Comment by dcarley** [comment]
The approach LGTM.

**Comment by mtekel** [comment]
Updated the commits.

**Comment by dcarley** [comment]
Will test after lunch

**Comment by dcarley** [comment]
:+1:
